ert3/config/_ensemble_config.py

```diff
@@ -31,12 +32,12 @@ class Input(_EnsembleConfig):
 
 class EnsembleConfig(_EnsembleConfig):
     forward_model: ForwardModel
-    input: Tuple[Input, ...]
+    inputs: Tuple[Input, ...] = Field(None, alias="input")
```

Comment:
I'm skeptical to the usage of `Fields` here. I see the motivation, but at the same time it sends developers and users down two different paths when it comes to terminology. Which I don't really think is worth it at this point. I would rather just change the variable name I think 🤷🏻

Reply:
I'm going to assume that this also implies for _stages_config